src/main/java/core/basesyntax/DataConverter.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverter {
+    public List<FruitTransaction> convert(List<String> rawData) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : rawData) {
+            String[] parts = line.split(",");
+            transactions.add(new FruitTransaction(parts[0], parts[1], Integer.parseInt(parts[2])));
```

Comment:
Ensure that the `FruitTransaction` class has a constructor that accepts three parameters: a string for the type, a string for the fruit, and an integer for the quantity. If the input format varies, consider adding validation or error handling for unexpected formats.